### DISCUSSION TITLE
Multicam Plugin - configure 2nd Webcam as IP camera

### DISCUSSION THREAD

**Discussion by svkz** [feelfreelinux/octo4a#297] [Q&A]
I´m installed Octo4a for fun, but i´m amazed of it. It´s more stable than using a RPI 4 and easy to use.

After installing the Multicam-Plugin i tried to configure an Iphone 4s, as an IP-camera. 

Porly, i did not have success using the frontend configuration.

Please help, to configure via SSH!

Thanks a lot
Smigel

**Comment by WriteCodeEveryday** [comment]
I'm in this one as well... I tried following an existing tutorial but I couldn't figure out how to get access to the /dev/

https://www.makersmashup.com/post/outside-the-box-multiple-webcam-setup-with-octoprint
